Command/HealthCheckCommand.php

```diff
@@ -51,18 +51,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         $runner->useAdditionalReporters($input->getOption('reporter'));
 
+        if (null !== $checkNames = $input->getArgument('checkNames')) {
+            $checkNames = explode(',', $checkNames);
```

Comment:
Hello dav,

Will "test2," and "test3," not be took has another arguments ?

Regards,
Le 21 avr. 2015 19:44, "dav-m85" <notifications@github.com> a écrit :

> In Command/HealthCheckCommand.php
> <https://github.com/lafourchette/LiipMonitorBundle/pull/3#discussion_r28803201>
> :
>
> > @@ -51,18 +51,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
> >          }
> >          $runner->useAdditionalReporters($input->getOption('reporter'));
> >
> > +        if (null !== $checkNames = $input->getArgument('checkNames')) {
> > +            $checkNames = explode(',', $checkNames);
>
> Please be more defensive here. I could provide "test1, test2, test3" which
> would evaluate as "test1", " test2", " test3", which is not what I want.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lafourchette/LiipMonitorBundle/pull/3/files#r28803201>
> .
>